server/utils.go

```diff
@@ -279,3 +280,24 @@ func getUlimitsFromConfig(config Config) ([]ulimit, error) {
 func translateLabelsToDescription(labels map[string]string) string {
 	return fmt.Sprintf("%s/%s/%s", labels[types.KubernetesPodNamespaceLabel], labels[types.KubernetesPodNameLabel], labels[types.KubernetesContainerNameLabel])
 }
+
+// Validate given hostIP IP belongs to the current host
+// adapted from github.com/kubernetes/kubernetes/pkg/kubelet/kubelet_node_status.go
+func validateHostIP(hostIP net.IP) error {
```

Comment:
I recommend to return a bool here if we do not do anything with the error message. The we could prefix the function with `is`, like `isValidHostIP`.